modules/javafx.web/src/test/java/test/javafx/scene/web/MiscellaneousTest.java

```diff
@@ -150,7 +152,9 @@ public void start() {
            submit(obj::start);
            // Try accessing the resultObject created in JavaFX Application Thread
            // from someother thread. It should throw an exception.
-           obj.resultObject.toString();
+           assertThrows(IllegalStateException.class, () -> {
```

Comment:
on second thought, I think this and the previous changes, even though not technically equivalent, are perfectly fine, in the spirit of the test.